src/domain/schema.graphql

```diff
@@ -0,0 +1,152 @@
+# A workaround for debugging.
+type Debug @entity {
+	id: ID!
+	message: String!
+}
+
+enum Outcome {
```

Comment:
should we add Rejected (preBoosted timeout)

Reply:
`preBoostedTimeout == Resolved && boostedAt == null`